purchase_stock_bl_date/readme/USAGE.rst

```diff
@@ -0,0 +1,15 @@
+1. Create or Edit existing quotation
+
+* Go to the purchase
+* Create new quotation or edit existing quotation
+* Select the date you wish on bl_date field under Other Information tab, then confirm the purchase order
+
+2. Validate the related stock picking
+
+* On purchase order, click "Receive product" button
+* On picking, make sure the bl_date has been reflected on the Other Information tab, then validate the picking
+
+3. Check computed amount on generated Journal Entry
+
+* Go to Invoicing> Accounting> Journal Entry
+* Open the stock valuation entry, make sure the amount calculated with the rate that applied on the specified bl_date
```

Comment:
Here is my version:

```suggestion
* Set B/L Date in purchase order as appropriate. When the purchase order is confirmed and the receipt picking is generated, the set date is passed over to the corresponding field of the picking.
* Adjust B/L Date of the receipt picking as necessary before validating the receipt.
```

Who is our audience? It isn't end users. Instead, it's ourselves, individuals with business experience and the capability to read the code. Let's be clear on this: We are writing for our future selves.

In this context, what we aim to explain here is the design concept rather than providing step-by-step instructions for end users. I understand that the README files of some OCA modules are written in this style. However, I've noticed that these documents often get neglected during design changes because verbosity increases the maintenance cost.